src/test/regress/expected/multi_sequence_default.out

```diff
@@ -127,30 +124,45 @@ SELECT * FROM seq_test_0_local_table ORDER BY 1, 2 LIMIT 5;
 ---------------------------------------------------------------------
  integer |     1 |       1 | 2147483647 |         1 | no      |     1
 
--- cannot change the type of a sequence used in a distributed table
--- even if metadata is not synced to workers
+-- cannot alter a sequence used in a distributed table
+-- since the metadata is synced to workers
 ALTER SEQUENCE seq_0 AS bigint;
-ERROR:  Altering a sequence used in a distributed table is currently not supported.
+ERROR:  Altering a distributed sequence is currently not supported.
 ALTER SEQUENCE seq_0_local_table AS bigint;
-ERROR:  Altering a sequence used in a local table that is added to metadata is currently not supported.
--- we can change other things like increment
--- if metadata is not synced to workers
+ERROR:  Altering a distributed sequence is currently not supported.
 ALTER SEQUENCE seq_0 INCREMENT BY 2;
```

Comment:
would be good to move these two lines in to the transaction block you added now and show that they work fine after `stop_metadata_Sync`

Reply:
I had to create a new sequence sequence to do this.